src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -12,29 +12,29 @@
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.event.Event;
-import seedu.address.model.person.Person;
+import seedu.address.model.participant.Participant;
 
 /**
  * An Immutable AddressBook that is serializable to JSON format.
  */
 @JsonRootName(value = "addressbook")
 class JsonSerializableAddressBook {
 
-    public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
+    public static final String MESSAGE_DUPLICATE_PERSON = "Participants list contains duplicate participant(s).";
```

Comment:
Might wanna change all instances of "MESSAGE_DUPLICATE_PERSON" to "MESSAGE_DUPLICATE_PARTICIPANTS" using Intellij.

Reply:
will do